cypress/support/pages/userFlow.pageObject.js

```diff
@@ -0,0 +1,11 @@
+import PageObject from '../PageObject';
+
+class UserFlowPageObject extends PageObject {
```

Comment:
What is the User Flow? I can't get what does this class represent. Consider store this action in commands or another POM class